CONTRIBUTING.md

```diff
@@ -234,8 +234,11 @@ A problem must have a unique slug. This slug is used as
 
 * Do the same as when [porting an
   exercise](#porting-an-exercise-to-another-language-track).
-  Reference the PR in problem-specifications if it hasn't been merged yet,
-  this **must not** be merged until the exercism/problem-specifications PR is merged.
+  Reference the PR in problem-specifications.
+  It's suggested but not required to wait until the problem-specifications PR is merged before merging the track-specific PR, for the following reasons:
```

Comment:
It's suggested, but not required, to wait ......

Reply:
the commas, right? okay